UnitedIncome/components
# QuestionAnswer Component - header can take node + string

### Description
> Please provide a brief description of your changes below.

Add ability to pass in a node in the header of the QuestionAnswer component.

### Testing Instructions
> What should be done in order to test this pull request, or re-create its effects.

Pass in a node in the header and make sure everything looks okay.

### Screenshots
> If this pull request includes a visual change please attach any applicable screenshots.

### Comments
> Any additional comments that will aid with the review process.
